symbols/ram.yml

```diff
@@ -191,6 +191,12 @@ arm9:
         2: Today's "Oh My Gosh"
         3: Here Comes Team Charm!
         4: In the Future of Darkness
+    - name: KAOMADO_STREAM
+      address:
+        NA: 0x22AB4B0
+      length:
+        NA: 0x48
+      description: The file stream utilized for all Kaomado portrait loads
```

Comment:
Is this of type `struct file_stream`, using, e.g., `FileOpen` and such? If so we should be more precise and note that here.